src/sentry/notifications/notifications/active_release.py

```diff
@@ -0,0 +1,205 @@
+from __future__ import annotations
+
+from typing import Any, Mapping, MutableMapping, Optional, Sequence, TypedDict
+from urllib.parse import quote, urlencode
+
+from sentry import features
+from sentry.models import Release, ReleaseActivity, ReleaseCommit, User
+from sentry.notifications.notifications.rules import AlertRuleNotification, logger
+from sentry.notifications.types import ActionTargetType, NotificationSettingTypes
+from sentry.notifications.utils import (
+    get_group_settings_link,
+    get_integration_link,
+    get_interface_list,
+    get_rules,
+    has_alert_integration,
+    has_integrations,
+)
+from sentry.plugins.base import Notification
+from sentry.types.integrations import EXTERNAL_PROVIDERS
+from sentry.types.releaseactivity import ReleaseActivityType
+from sentry.utils import metrics
+from sentry.utils.http import absolute_uri
+
+
+class ActiveReleaseAlertNotification(AlertRuleNotification):
```

Comment:
I believe the only that changed was overriding the `send()` method from `AlertRuleNotification`, and line 28 where we changed it to respect the new notification setting type. 

Still need to update that file to remove any references of any Rules since this notification type isn't 'aware' of any Rules.